agent/engine/docker_task_engine.go

```diff
@@ -103,10 +103,11 @@ const (
 	fluentTagDockerFirelensV2Format = "%s.%s"
 
 	// Environment variables are needed for firelens
-	fluentNetworkHost      = "FLUENT_HOST"
-	fluentNetworkPort      = "FLUENT_PORT"
-	FluentNetworkPortValue = "24224"
-	FluentAWSVPCHostValue  = "127.0.0.1"
+	fluentNetworkHost        = "FLUENT_HOST"
+	fluentNetworkPort        = "FLUENT_PORT"
+	FluentNetworkPortValue   = "24224"
+	AWSVPCHostValue          = "127.0.0.1"
```

Comment:
Are these values expected to change over time? 
Why not inject these values in ECS backend instead of agent doing it?

Reply:
So for AWSVPC and Host mode, the values are always the same. Its bridge mode where Agent code is needed, since it needs to inject the bridge IP. 

Therefore, I think its simpler to just have the agent manage all the values.

Reply:
If you disagree with that, it's totally doable for us to inject the static values for host and awsvpc on the backend. And then agent only injects the value for bridge. I think separate which component injects based on the network mode is weird tho.

Reply:
I see, I'm ok with keeping this in agent for now.